irods/test/data_obj_test.py

```diff
@@ -2955,6 +2955,53 @@ def test_replica_truncate__issue_534(self):
                     if data_objs.exists(data_path):
                         data_objs.unlink(data_path, force=True)
 
+    def test_get_collection__issue_734(self):
+        from irods.helpers import get_collection, get_data_object
+
+        sess = self.sess
+        logical_path = "{}/{}".format(
+            self.coll_path, unique_name(my_function_name(), datetime.now())
+        )
+        sess.collections.create(logical_path)
+
+        self.assertIs(get_data_object(sess, logical_path), None)
+        self.assertTrue(get_collection(sess, logical_path))
```

Comment:
It's not clear to me what this test is about ... wait, is it testing the utility functions?

If so, consider using more words - e.g. `test_get_collection_returns_X_when_Y__issue_734`.

Reply:
Think what's there now is better?

Reply:
I agree.